src/components/transactions/TxSummary/index.tsx

```diff
@@ -52,15 +52,18 @@ const TxSummary = ({ item, isGrouped }: TxSummaryProps): ReactElement => {
     : undefined
 
   const displayConfirmations = isQueue && !!submittedConfirmations && !!requiredConfirmations
+  const displayInfo = !tx.txInfo.richDecodedInfo && tx.txInfo.type !== TransactionInfoType.TRANSFER
 
   return (
     <Box
       className={`${css.gridContainer} ${
         isQueue
-          ? nonce && !isGrouped
+          ? displayInfo
             ? css.columnTemplate
-            : css.columnTemplateWithoutNonce
```

Comment:
These 2 classes were identical so I removed the latter one. Maybe they were needed at some point?